recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "rapidjson" %}
-{% set version = "1.1.0" %}
-{% set sha256 = "bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e" %}
+{% set version = "2024.4.9" %}
+{% set commit_hash = "ab1842a2dae061284c0a62dca1cc6d5e7e37e346" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/miloyip/rapidjson/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/Tencent/rapidjson.git
```

Comment:
Let's please use GitHub tarballs and not plain git clones for this.

Reply:
I thought that was only available for tags, but of course it works for commits too. Fixed now.